minisforum/v3/src.nix

```diff
@@ -0,0 +1,8 @@
+# Outsource for aspm_v3.sh and dsdt.dsl
+{ fetchFromGitHub, ... }:
+fetchFromGitHub {
```

Comment:
Any comments on the license of this repository? This might be important for some people.

Reply:
Which licence should I add?
The `aspm_v3.sh` already has something from the original author,
`dsdt.dsl` has been extracted from my machine.

Reply:
`aspm_v3.sh` looks like MIT. If you "extract" as in generated the second file than it would be "public domain" since it doesn't have an author. The file has an Intel Copyright, but I assume this comes from the disassembler? I don't think intel really can claim any license here if it was generated, in the same way adobe cannot claim copyright over photos edited with photoshop. So I think MIT is fine for the repository.